### DISCUSSION TITLE
How to get search_issues() returns all issues ?

### DISCUSSION THREAD

**Discussion by nyue** [pycontribs/jira#1827] [Q&A]
```python
issues = jira.search_issues('')
for singleIssue in issues:
  print(singleIssue)
```

Maybe I am not understanding jql, I was thinking that jql constraints the search and if we provide an empty string, I should expect search_issues() to return all the issues a given Jira server instance knows about.

**Comment by adehad** [comment]
From the UI based JQL search, I would think the same, but perhaps you can explore more generic JQL e.g.
`ORDER BY created DESC` and see if that p
